pkg/bpf/map.go

```diff
@@ -114,6 +135,42 @@ type MapInfo struct {
 	OwnerProgType ProgType
 }
 
+// DesiredAction is the action to be performed on the BPF map
+type DesiredAction int
+
+const (
+	// OK indicates that to further action is required and the entry is in
+	// sync
+	OK DesiredAction = iota
+
+	// Insert indicates that the entry needs to be created or updated
+	Insert
+
+	// Delete indicates that the entry needs to be deleted
+	Delete
+)
+
+func getActionName(action DesiredAction) string {
```

Comment:
Could you switch this to be the `String` function on type DesiredAction?

Reply:
done